src/functions/hideoutFunctions.js

```diff
@@ -0,0 +1,56 @@
+import hideoutData from '../../tarkovdata/hideout.json'
+
+export default {
```

Comment:
Already doing better than me about organizing the code 👍

Reply:
I was also trying to introduce project-wide constants for the gameEdition, but for some reason, Vue wasn't accepting any of the ways I found in guides online. I gave up probably too soon.